presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -204,48 +206,50 @@ public void printFinalInfo(Optional<Long> clientStopTimestamp)
-                    formatCountRate(stats.getProcessedRows(), cpuTime, false),
-                    formatDataRate(bytes(stats.getProcessedBytes()), cpuTime, true),
-                    (int) percentage(stats.getCpuTimeMillis(), stats.getWallTimeMillis()));
-            out.println(cpuTimeSummary);
-
-            double parallelism = cpuTime.getValue(MILLISECONDS) / serverSideWallTime.getValue(MILLISECONDS);
-
-            // Per Node: 3.5 parallelism, 83.3K rows/s, 0.7 MB/s
-            String perNodeSummary = format("Per Node: %.1f parallelism, %5s rows/s, %8s",
-                    parallelism / nodes,
-                    formatCountRate((double) stats.getProcessedRows() / nodes, serverSideWallTime, false),
-                    formatDataRate(bytes(stats.getProcessedBytes() / nodes), serverSideWallTime, true));
-            reprintLine(perNodeSummary);
-
-            // Parallelism: 5.3
-            out.println(format("Parallelism: %.1f", parallelism));
-
-            // Peak User Memory: 1.97GB
-            reprintLine("Peak User Memory: " + formatDataSize(bytes(stats.getPeakMemoryBytes()), true));
-            // Peak Total Memory: 1.98GB
-            reprintLine("Peak Total Memory: " + formatDataSize(bytes(stats.getPeakTotalMemoryBytes()), true));
-            // Peak Task Total Memory: 1.99GB
-            reprintLine("Peak Task Total Memory: " + formatDataSize(bytes(stats.getPeakTaskTotalMemoryBytes()), true));
-
-            // Spilled Data: 20GB
-            if (stats.getSpilledBytes() > 0) {
-                reprintLine("Spilled: " + formatDataSize(bytes(stats.getSpilledBytes()), true));
-            }
+            if (runtime) {
+                // CPU Time: 565.2s total,   26K rows/s, 3.85MB/s
+                Duration cpuTime = millis(stats.getCpuTimeMillis());
+                String cpuTimeSummary = format("CPU Time: %.1fs total, %5s rows/s, %8s, %d%% active",
+                        cpuTime.getValue(SECONDS),
+                        formatCountRate(stats.getProcessedRows(), cpuTime, false),
+                        formatDataRate(bytes(stats.getProcessedBytes()), cpuTime, true),
+                        (int) percentage(stats.getCpuTimeMillis(), stats.getWallTimeMillis()));
+                out.println(cpuTimeSummary);
+
+                double parallelism = cpuTime.getValue(MILLISECONDS) / serverSideWallTime.getValue(MILLISECONDS);
+
+                // Per Node: 3.5 parallelism, 83.3K rows/s, 0.7 MB/s
+                String perNodeSummary = format("Per Node: %.1f parallelism, %5s rows/s, %8s",
+                        parallelism / nodes,
+                        formatCountRate((double) stats.getProcessedRows() / nodes, serverSideWallTime, false),
+                        formatDataRate(bytes(stats.getProcessedBytes() / nodes), serverSideWallTime, true));
+                reprintLine(perNodeSummary);
+
+                // Parallelism: 5.3
+                out.printf("Parallelism: %.1f", parallelism);
```

Comment:
Why the change form println to print without newline?

Reply:
Oops, missed the format. I was returning it to the original version.